components/notification/NotificationPaginator.vue

```diff
@@ -13,6 +13,14 @@ const virtualScroller = false // TODO: fix flickering issue with virtual scroll
 
 const groupCapacity = Number.MAX_VALUE // No limit
 
+function includeNotificationsForStatusCard({ type, status }: mastodon.v1.Notification) {
+  // Exclude update, mention, pool and status notifications without the status entry:
+  // no makes sense to include them
+  // Those notifications will be shown using StatusCard SFC:
+  // check NotificationCard SFC L68 and L81 => :status="notification.status!"
+  return status || !['update', 'mention', 'poll', 'status'].includes(type)
```

Comment:
I think it would be good to extract this array out from the function

Reply:
done

Reply:
@patak-dev don't forget this one :pray: